Golem/InvoiceEventsLoop.cs

```diff
@@ -20,47 +21,81 @@ public InvoiceEventsLoop(YagnaApi yagnaApi, CancellationToken token, ILogger log
         _jobs = jobs;
     }
 
-    public async Task Start()
+    public Task Start()
     {
-        _logger.LogInformation("Starting monitoring invoice events");
+        return Task.WhenAll(PaymentsLoop(), InvoiceLoop());
+    }
 
-        DateTime newReconnect = DateTime.Now;
+    public async Task InvoiceLoop()
+    {
+        _logger.LogInformation("Starting monitoring invoice events");
 
         await Task.Yield();
-        
+        DateTime since = DateTime.Now;
+
         while (!_token.IsCancellationRequested)
         {
             try
             {
-                var invoiceEvents = await _yagnaApi.GetInvoiceEvents(_since, _token);
```

Comment:
We have been previously always getting invoices since `DateTime.MinValue;`, and this change is on purpose? (just checking)

Reply:
no